paymentsheet/src/test/java/com/stripe/android/paymentelement/embedded/content/EmbeddedContentScreenshotTest.kt

```diff
@@ -34,10 +35,30 @@ internal class EmbeddedContentScreenshotTest {
     @Test
     fun displaysVerticalModeListWithMandate() {
         val metadata = PaymentMethodMetadataFactory.create()
-        val interactor = FakePaymentMethodVerticalLayoutInteractor.create(metadata)
+        val interactor = FakePaymentMethodVerticalLayoutInteractor.create(
+            paymentMethodMetadata = metadata,
+            mandate = "Some mandate".resolvableString,
+        )
         val content = EmbeddedContent(
             interactor = interactor,
+            embeddedViewDisplaysMandateText = true,
+            rowStyle = PaymentSheet.Appearance.Embedded.RowStyle.FloatingButton.default
+        )
+        paparazziRule.snapshot {
+            content.Content()
+        }
+    }
+
+    @Test
+    fun displaysVerticalModeListWithoutMandate() {
```

Comment:
Moved a tiny bit more logic to the UI, so I added this test for it. We have a mandate, but `embeddedViewDisplaysMandateText` is false.